plugins/contactpressuresensorsarray/include/yarp/dev/ContactPressureSensorsArrayDriver.h

```diff
@@ -0,0 +1,225 @@
+ *
+ * Parameters accepted in the config argument of the open method for [DRIVER]:
+ * 
+ * | Parameter name|      Type     |Units|Default Value|Required |                       Description                      |                        Notes                     |
+ * |:-------------:|:-------------:|:---:|:-----------:|:-------:|:------------------------------------------------------:|:------------------------------------------------:|
+ * | device        |    string     |  -  |    -        |   Yes   | name of the device driver to instantiate               | MUST be set to gazebo_contactpressuresensorsarray|
+ * | loadCellNames |list of strings|  -  |    -        |   Yes   | list of load cell names, eg. (lr rr lf rf)             | locations MUST be obtained manually from urdf/sdf|
+ * | loadCellX     |list of doubles|  m  |    -        |   Yes   | list of load cell location X coordinate wrt link origin| MUST be the same size and order as loadCellName  |
+ * | loadCellY     |list of doubles|  m  |    -        |   Yes   | list of load cell location Y coordinate wrt link origin| MUST be the same size and order as loadCellName  |
+ * | loadCellZ     |list of doubles|  m  |    -        |   Yes   | list of load cell location Z coordinate wrt link origin| MUST be the same size and order as loadCellName  |
+ * | linkName      |     string    |  -  |    -        |   Yes   | name of the link associated to the contact sensor      |                            -                     |
+ *
+ * NOTE: Although being sensor-related, this plugin is implemented as a model plugin instead of a sensor plugin. This design choice is due to the fact that transformations related
+ * to the link were required for computations in this device; which could not be achieved through a sensor plugin, since Gazebo inherently divides its
+ * physics engine and sensor engine in its software architecture. As a model plugin, the link information can be directly accessed,
+ * however the sensor name and collision name related to the link are obtained by parsing through the SDF directly. Again, this design decision
+ * of parsing through the sdf is made due to the fact that the Link information does not give the type of sensors but only sensor counts.
+ * 
+ * This plugin in its current version assumes a only single collision element is associated to the specified link
+ * This sensor plugin in its current version handles only one link with a plugin instance 
+ * (i.e., multiple links cannot be specified per plugin, but multiple plugins can be instantiated through the sdf)
+ * The collision element in the contact sensor added to the specific link in the sdf should be set to {link_name}_collision
+ * The name of the collision element associated to the link should be set to default (i.e. "name" parameter should not be set so that it can take
+ * the default name {link_name}_collision. Otherwise, things become worse.)
```

Comment:
Not a native speaker, but I don't think you can close the a period inside a bracket.

Reply:
Noted. Will push this change along with some major commit.